x/evm/keeper/grpc_query.go

```diff
@@ -675,19 +670,20 @@ func (k *Keeper) traceMsg(
 		}
 	}()
 
-	var stateOverrides rpctypes.StateOverride
 	if traceConfig.StateOverrides != nil {
 		config, err := json.Marshal(traceConfig.StateOverrides)
 		if err != nil {
 			return nil, 0, status.Error(codes.InvalidArgument, err.Error())
 		}
 
-		if err := json.Unmarshal(config, &stateOverrides); err != nil {
+		if err := json.Unmarshal(config, cfg.Overrides); err != nil {
```

Comment:
```suggestion
		if err := json.Unmarshal(config, &cfg.Overrides); err != nil {
```

Reply:
`cfg.Overrides` is a pointer already.

Reply:
actually why we have json marshaling and unmarshalling at the same place, maybe you can fix that in your pr.

Reply:
it's pointer of pointer, no need init, but marshaling and unmarshalling here might just to ensure marshal no error?

Reply:
I think it's simply to convert protobuf msg to custom struct

Reply:
seems need init Overrides

Reply:
should has fixed